airbyte-integrations/connectors/source-freshsales/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_freshsales ./source_freshsales
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.3
+LABEL io.airbyte.version=0.1.4
```

Comment:
Just refactors as far as I can see, as well as declaring undeclared columns (column selection hasn't been introduced yet, so this doesn't affect end user experince) - patch seems good here